site/miniHeader.less

```diff
@@ -2,7 +2,7 @@
   background: #435052;
   position: fixed;
   width: 100%;
-  z-index: 1;
+  z-index: 100;
```

Comment:
The owner height example was allowing the page to be displayed over some parts of the header (but not all).  This fixes that by giving the header a higher z-index.